workspaces/spectacle/tap-snapshots/test-index.ts-TAP.test.js

```diff
@@ -100,17 +100,7 @@ exports[
 Object {
   "data": Null Object {
     "endpointChanges": Null Object {
-      "endpoints": Array [
```

Comment:
This is a change so that specifying `sinceBatchCommitId: undefined` will return an empty list of changes, rather than every change that ever happened

Reply:
Ok, apparently our changelog feature was relying on sending `sinceBatchCommitId: undefined` to return us every change that happened in the spec. I don't think this makes sense (because the results are, everything changed) - but to reduce the amount of work, I'm just going to leave it as is.